### DISCUSSION TITLE
Use TLS for some subjects and plaintext for others?

### DISCUSSION THREAD

**Discussion by C-Lunn** [nats-io/nats-server#6116] [Q&A] [answered]
Hi all,

I'd like to know if it's possible to enable TLS certificate authentication for some subjects, and plaintext for all others?

I can't seem to find anything in the docs, but I know it is possible to accept both types of connection on the same port (https://github.com/nats-io/nats-server/issues/1541).

Many thanks.

**Comment by wallyqs** [comment, marked as answer]
Hi, it is not possible to accept both types of auth in the same server.
